test/controllers/airlines_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AirlinesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
I recommend that you please remove all unnecessary files, which serve nothing to your codebase. That way your code can be easily read & look more attractive.